robotis-op2/using-remote-control-from-robot-window.md

```diff
@@ -26,9 +26,9 @@ remote control has been sucessfully started.
 You can now easily start and pause
 your controller in remote control mode by using the run and pause buttons of
 Webots simulation (see chapter about [examples](examples.md) for more details).
-Warning: if you revert
-the simulation it will stop the remote control mode. In order to stop the
-remote control (without reverting) simply press the following button:
+Warning: if you reload
+the world it will stop the remote control mode. In order to stop the
+remote control (without reloading) simply press the following button:
```

Comment:
One sentence per line.

Reply:
Do we really want to introduced so many changes for non-new sentences, because it will hopefully be done automatically in #587 and introduce a lot of conflicts that we will have to manually manage if we changes all the paragraph that doesn't respect the rule now?